add-on/panels/js/roomStore.js

```diff
@@ -544,7 +550,8 @@ loop.store = loop.store || {};
      */
     openRoom: function(actionData) {
       loop.requestMulti(
-        ["Rooms:Open", actionData.roomToken],
+        ["OpenURL", TOC_URL + actionData.roomToken],
+        ["LoadSidebar", actionData.roomToken],
```

Comment:
I think that would probably be fine. There's a chance with moving to a dedicated room that we'll also need to have some sort of sidebar open method, but until we've worked out what we're doing its hard to say.

So yes, doing the LoadSidebar from LoopRooms#open, seems reasonable, we can always move it again later if necessary.